src/convertToObject.js

```diff
@@ -7,6 +7,23 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arr = sourceString.split(';');
+  let aux;
+  let ky;
+  let vl;
```

Comment:
quick note: 
**key** and **val** are generally better naming conventions than this

generally try maintaining the first (or the most meaningful) 3 to 5 letters of the word that you want to shorten, like cellphone -> cel, address -> addr